server/src/controllers/api.rs

```diff
@@ -13,6 +13,17 @@ pub struct QueryString {
     name: String,
 }
 
+#[derive(Serialize, Deserialize)]
+struct ScoreboardEntry {
+    player_id: u32,
```

Comment:
Can we expose more data? how about name, sth like that. We can't show the data just the id

Reply:
Thanks, I just pushed my refactor to include more info (`api_key`, `team_name`) in scoreboard response.